src/main/java/org/ambraproject/wombat/controller/ArticleController.java

```diff
@@ -311,28 +321,14 @@ public Object receiveNewComment(HttpServletRequest request,
                                   @RequestParam("commentTitle") String commentTitle,
                                   @RequestParam("comment") String commentBody,
                                   @RequestParam("isCompetingInterest") boolean hasCompetingInterest,
-                                  @RequestParam(value = "authorEmailAddress", required=false) String authorEmailAddress,
-                                  @RequestParam(value = "authorName", required=false) String authorName,
-                                  @RequestParam(value = "authorPhone", required=false) String authorPhone,
-                                  @RequestParam(value = "authorAffiliation", required=false) String authorAffiliation,
```

Comment:
authorPhone and authorAffiliation gets deleted.

Reply:
Didn't catch this, must have been a bad merge on my part. Thanks.